README.md

```diff
@@ -14,21 +14,35 @@ This repository follows the [RealWorld](https://github.com/gothinkster/realworld
 
 ## Installation
 
-You can install and build the project with:
+Firstly, clone the repository:
 
 ```sh
-# clone the repository
 git clone https://github.com/thomashoneyman/purescript-halogen-realworld
 cd purescript-halogen-realworld
+```
+
+In order to install the JavaScript dependencies:
 
-# install dependencies
+```sh
 yarn install
+```
+
+Then, build the project (this command will perform a `spago build`):
```

Comment:
Perhaps here we could note "(this command will run `spago build`; see the [package.json](...) file to see all helper scripts for the project)" so people know why this command runs spago build, and the others do what they do?

Reply:
Yes, good idea. I'll remove the other comments regarding which yarn command performs what real command.

As a sidenote: did you intend for the _Resources_ and _Contributing_ headlines to be of another level than `##`?

Reply:
I intended _Resources_ to be a sub-section of _Learning PureScript_, but _Contributing ought to be `##`. Good catch!

Reply:
OK, all fixed!